src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/cardgrid/viewmodel/CardGridViewModel.kt

```diff
@@ -0,0 +1,103 @@
+package org.wycliffeassociates.otter.jvm.app.ui.cardgrid.viewmodel
+
+import com.github.thomasnield.rxkotlinfx.changes
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import com.github.thomasnield.rxkotlinfx.toObservable
+import io.reactivex.Observable
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.collections.FXCollections
+import javafx.collections.ObservableList
+import org.wycliffeassociates.otter.common.data.model.ContentLabel
+import org.wycliffeassociates.otter.common.data.workbook.Chapter
+import org.wycliffeassociates.otter.common.data.workbook.Workbook
+import org.wycliffeassociates.otter.jvm.app.ui.cardgrid.CardData
+import org.wycliffeassociates.otter.jvm.app.ui.cardgrid.CardDataMapper
+import org.wycliffeassociates.otter.jvm.app.ui.workbook.viewmodel.WorkbookViewModel
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import tornadofx.*
+
+class CardGridViewModel: ViewModel() {
+    private val workbookViewModel: WorkbookViewModel by inject()
+
+    // List of content to display on the screen
+    // Boolean tracks whether the content has takes associated with it
+    val allContent: ObservableList<CardData>
+            = FXCollections.observableArrayList()
+    val filteredContent: ObservableList<CardData>
+            = FXCollections.observableArrayList()
+
+    // Whether the UI should show the plugin as active
+    private var showPluginActive: Boolean by property(false)
+    val showPluginActiveProperty = getProperty(CardGridViewModel::showPluginActive)
```

Comment:
These are unused, and are maybe out of place?

Reply:
Not sure why they were there. I renamed this file but git didn't recognize it as just a rename apparently. They do appear to be out of place, though- there is a property with the same name in TakeManagementViewModel (in later PR's it is in RecordableViewModel since TMVM got deleted). I just removed this property from CardGridViewModel since it is unused here.